src/cryptography/hazmat/primitives/hashes.py

```diff
@@ -148,6 +149,15 @@ class MD5(object):
     digest_size = 16
     block_size = 64
 
+    def __init__(self):
```

Comment:
I don't think we need to use `__init__`, `utils.deprecated()` should cover this.

Reply:
OK, will fix